6_visualize.yml

```diff
@@ -27,19 +27,18 @@ targets:
   # into an RDS file with a corresponding .rds.ind indicator file.
 
   # right here we will create and run a task table that will be responsible for generating
-  # the components, e.g., '6_visualize/out/style/storm_point_20170926_11.rds.ind' through
-  # '6_visualize/out/style/streamdata_20170926_11.rds.ind', one per column=step. and then the final
+  # the components, e.g., '5_vizprep/out/storm_point_20170926_11.rds.ind' through
+  # '5_vizprep/out/streamdata_20170926_11.rds.ind', one per column=step. and then the final
   # column=step of that task table will be to run create_storm_frame on the output of all the other
   # steps in that row=task=timestep.
 
   # here's an example of the a single task-step for that last column=step of the task table:
-  6_visualize/out/storm_frame .png:
+  6_visualize/out/storm_frame.png:
     command: create_storm_frame(
       png_file=target_name, config=storm_frame_config,
-      '6_visualize/out/style/basemap.rds.ind',
-      '6_visualize/out/style/focus_geoms.rds.ind',
-      '6_visualize/out/style/secondary_geoms.rds.ind',
-      '6_visualize/out/style/storm_line.rds.ind',
-      '6_visualize/out/style/storm_point_20170926_11.rds.ind',
-      '6_visualize/out/style/precip_raster_20170926_11.rds.ind',
-      '6_visualize/out/style/streamdata_20170926_11.rds.ind')
+      '5_vizprep/out/view_fun.rds.ind',
+      '5_vizprep/out/basemap_fun.rds.ind',
+      '5_vizprep/out/storm_line_fun.rds.ind',
+      '5_vizprep/out/storm_point_-20170826_11-_fun.rds.ind')
```

Comment:
no good just using a dash to separate the days from the hours, huh? e.g., `5_vizprep/out/storm_point_20170826-11_fun.rds.ind`? that's how i would have done this, to preserve the ability to really easily parse the string with a strsplit on `_`. what's your standard code for parsing with `[` and `]` in some pieces of the filename?

Reply:
I want it to use `[20170826-11]` and switched that to `-` and `_` as a placeholder

Reply:
to get what is within the first set of brackets:
```r
gsub("\\-|\\-", "", regmatches(filename, gregexpr("\\-.*?\\-", filename))[[1]][1])
```

Reply:
thanks